generators/lib/DnnGeneratorBase.js

```diff
@@ -187,9 +187,17 @@ module.exports = class DnnGeneratorBase extends Generator {
       // TODO: Remove the ones that don't matter
       if (fs.existsSync("C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\MSBuild\\Microsoft\\VisualStudio\\v16.0\\WebApplications\\Microsoft.WebApplication.targets") ||
         fs.existsSync("C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Professional\\MSBuild\\Microsoft\\VisualStudio\\v16.0\\WebApplications\\Microsoft.WebApplication.targets") ||
-        fs.existsSync("C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v16.0\\WebApplications\\Microsoft.WebApplication.targets")) {
+        fs.existsSync("C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v16.0\\WebApplications\\Microsoft.WebApplication.targets")||
+        fs.existsSync("C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\MSBuild\\Microsoft\\VisualStudio\\v16.0\\WebApplications\\Microsoft.WebApplication.targets")) {
```

Comment:
@VAlmea There appears to be a duplicate line of code, right?  Did you intend to add VS 2022?

Reply:
yes. I tried to use VS 2022

Reply:
@VAlmea Any answer?

Reply:
@WillStrohl This code is not duplicated. The last line of code refers only to the BuildTools package for vs 2019. This works in the case of having vs 2022 and not having 2019 installed. It only verifies that there is the BuildTools package for vs 2019 without having to have VS 2019 installed